web/packages/teleport/src/AuthConnectors/AuthConnectors.tsx

```diff
@@ -52,13 +52,20 @@ export function AuthConnectors(props: State) {
 
   return (
     <FeatureBox>
-      <FeatureHeader>
+      <FeatureHeader
+        css={`
+          @media screen and (max-width: 800px) {
+            flex-direction: column;
+            height: auto;
+            gap: 10px;
+            margin: 0 0 10px 0;
+            padding-bottom: 10px;
+            justify-content: center;
+          }
+        `}
```

Comment:
Would 

```
const ResponsiveFeatureHeader = styled(FeatureHeader)`
  @media screen and (max-width: 800px) {
    flex-direction: column;
    height: auto;
    gap: 10px;
    margin: 0 0 10px 0;
    padding-bottom: 10px;
    justify-content: center;
  }
`;
```

be better here? I don't have a strong preference, I just try to avoid having `css` blocks in the layout